### ISSUE TITLE
Accessibility per UC: need a link in the footer

### ISSUE THREAD

**Issue by MomoMoses** [johnjung/bmrcportal#172]
UChicago guidelines state that there should be a link to the Accessibility page at UC in the footer of the website. 
[link to be used](https://accessibility.uchicago.edu/)
